app/controllers/api/v2/moves_actions.rb

```diff
@@ -166,7 +166,7 @@ def create_event(move)
         recorded_at: now,
         notes: 'Automatically generated event',
         details: {},
-        supplier_id: move.supplier_id,
+        supplier_id: doorkeeper_application_owner&.id,
```

Comment:
the field `supplier_id` sounds like it is misnamed: 
* is it the supplier who will perform the move?
* the user (which may or may not be a supplier) who created the move?
if the latter, then a different name might be more appropriate (e.g. created_by)

Reply:
We're going to merge this and fix this after we know the frontend user token that created the event as part of the `created_by` attribute that we intend to add to all events later on.

Reply:
Had a conversation with Martyn on a call just now.